### PR TITLE
Changed repository to write documentation to biocro/BioCro-documentation

### PR DESCRIPTION
**Pull request by gsrohde**
Also: use `main` rather than `master` as the branch where pushes generate new documentation.

---

This is a stop-gap fix to get documentation up and running at the new location (under the `biocro` organization).

I'm temporarily using a landing page to ease the transition.

The old ebimodeling BioCro documentation page now points to that landing page.

If you don't like https://biocro.github.io/BioCro-documentation/ as the location, we can easily change it.  I considered just https://biocro.github.io/documentation/, thinking the extra "biocro" was redundant, but I wasn't sure if there would be other, separate documentation for other repositories under `biocro/`.  I used "BioCro-documentation" rather than "biocro-documentation" to distinguish between BioCro, the R package, and biocro, the organization; but maybe this is too fussy and will lead to unnecessary mistypings and 404 errors.  (The hostname portion of a URL is, in general, not case-sensitive, but the rest usually _is_ case sensitive.)

### CODE REVIEW

**Comment by eloch216** [comment]
Looks good to me! I don't have strong feelings about the address for the documentation, as long as it's stable.

> The old ebimodeling BioCro documentation page now points to that landing page.

Nice. I think there is still some old documentation publicly available from two repositories, though:
- https://github.com/ebimodeling/biocro-documentation
- https://github.com/ebimodeling/biocro-dev-documentation

**Comment by gsrohde** [comment]
> Looks good to me! I don't have strong feelings about the address for the documentation, as long as it's stable.

My intention is that https://biocro.github.io/BioCro-documentation/ will be the official location for the BioCro documentation, or at least the automatically-generated documentation.

My personal preference would be for https://biocro.github.io to be the official, published site for BioCro information, with, for now, that page just showing a set of links to the documentation for various versions (for now, a set of size one).  It wouldn't be hard to set this up, making a repository at biocro/biocro.github.io with just a single page for now.  It would also spare me the work of making the pkgdown home page appear at https://biocro.github.io/BioCro-documentation/ rather than at https://biocro.github.io/BioCro-documentation/v3.0.0/pkgdown/ or https://biocro.github.io/BioCro-documentation/latest/pkgdown/.  But you seemed to be concerned about users having to do an extra click to get there.

> 
> > The old ebimodeling BioCro documentation page now points to that landing page.
> 
> Nice. I think there is still some old documentation publicly available from two repositories, though:
> 
> * https://github.com/ebimodeling/biocro-documentation

I was planning to remove everything from this repository except for the page directing users to the new documentation site.  But I was waiting until I got sign off on this PR and merged it.

If there are no objections, I will do that now and then remove most of https://github.com/ebimodeling/biocro-documentation.

> * https://github.com/ebimodeling/biocro-dev-documentation
> 
> And at this link: https://ebimodeling.github.io/biocro-dev-documentation/

I wasn't worrying about these sites since I thought it was clear that these were for biocro-dev, not biocro.  But maybe that's only clear to us, and I should just go ahead and remove the https://github.com/ebimodeling/biocro-dev-documentation repository to avoid any potential confusion.  Or, I could replace what's there with a link to the new BioCro documentation site.

**Comment by justinmcgrath** [comment]
I think this is good to merge.

I don't see a reason to keep any of the old sites and redirect. We aren't so popular that anyone will have hard linked to those places, and if anyone did, it's certainly someone we personally know and we can sort it out. I don't like keeping having things spread out over multiple places. It creates confusion for people searching and a maintenance burden.

**Comment by justinmcgrath** [comment]
In the other thread, I think we've concluded that the version in DESCRIPTION here should be changed to 3.0.1 before merging.

**Comment by gsrohde** [comment]
> In the other thread, I think we've concluded that the version number in DESCRIPTION here should be changed to 3.0.1 before merging.

I've changed the version number and added to NEWS.md.  If you'd give this a quick final lookover, then I'll merge and tag this as a new release.

**Review by justinmcgrath — APPROVED** [review]

**Comment by gsrohde** [comment]
Actually, I'm thinking the trigger for running this workflow should be publication of a release rather than adding a release tag.

@justinmcgrath , when you publish a release, do you do that through the GitHub Web site?

**Comment by justinmcgrath** [comment]
I've used the Web site, and I thought that was the only way to make one, since they're a GitHub thing and not a git thing. When I've done it, I gave the tag name while making the release, and it made them both at the same time.

**Comment by eloch216** [comment]
> If there are no objections, I will do that now and then remove most of https://github.com/ebimodeling/biocro-documentation.
> 
> > * https://github.com/ebimodeling/biocro-dev-documentation
> > 
> > And at this link: https://ebimodeling.github.io/biocro-dev-documentation/
> 
> I wasn't worrying about these sites since I thought it was clear that these were for biocro-dev, not biocro. But maybe that's only clear to us, and I should just go ahead and remove the https://github.com/ebimodeling/biocro-dev-documentation repository to avoid any potential confusion. Or, I could replace what's there with a link to the new BioCro documentation site.

I don't think that will be clear to most people. They will just "BioCro" all over the page or repository, and assume that they are looking at BioCro documentation. I don't think people will immediately recognize that "biocro-dev" is a different thing.

> I don't see a reason to keep any of the old sites and redirect. We aren't so popular that anyone will have hard linked to those places, and if anyone did, it's certainly someone we personally know and we can sort it out. I also don't like keeping having things spread out over multiple places. It creates confusion for people searching and a maintenance burden.

I am also partial to this viewpoint.

**Comment by justinmcgrath** [comment]
Was this supposed to fix the link in the README? It still says it's not found. Maybe the README also needs to be updated?

**Comment by gsrohde** [comment]
> Was this supposed to fix the link in the README? It still says it's not found. Maybe the README also needs to be updated?

Yes, the README needs to be updated.  I'm currently working on a PR to change all references to "ebimodeling" to "biocro" (along with some other appropriate changes).  In the meantime, I'll set up an automatic redirect from https://ebimodeling.github.io/biocro-documentation/master/pkgdown/index.html to the new site as part of cleaning out the `ebimodeling/biocro-documentation` repository, since this is easy to do, doesn't require changes to `biocro/biocro`, a new PR, a new version, and all that.

By the way, I've already set up a redirect from https://biocro.github.io/ to https://biocro.github.io/BioCro-documentation/latest/pkgdown/, and I think the former could be the published location for the new documentation.  Right now, https://biocro.github.io/ takes one directly to the latest version of the pkgdown site, but it could eventually, if we wanted, be the location of a more complete, curated web site for BioCro, analogous to the [PEcAn project page](https://pecanproject.github.io/).

**Comment by gsrohde** [comment]
> I've used the Web site, and I thought that was the only way to make one, since they're a GitHub thing and not a git thing. When I've done it, I gave the tag name while making the release, and it made them both at the same time.

I guess there is a way of doing this from the command line (see https://stackoverflow.com/a/65474194), but I see no reason not to just use the Web site.

**Comment by gsrohde** [comment]
> In the meantime, I'll set up an automatic redirect from https://ebimodeling.github.io/biocro-documentation/master/pkgdown/index.html to the new site as part of cleaning out the `ebimodeling/biocro-documentation` repository, since this is easy to do, doesn't require changes to `biocro/biocro`, a new PR, a new version, and all that.

This is done.  Both the link to [BioCro Documentation Web Site](https://ebimodeling.github.io/biocro-documentation/master/pkgdown/index.html) and to [page that documents all of the quantities used by the Standard BioCro Module Library](https://ebimodeling.github.io/biocro-documentation/quantity_docs/quantities.html) under the _Software Documentation_ section of the README now get you to the right place.

**Comment by gsrohde** [comment]
> I don't think that will be clear to most people. They will just see "BioCro" all over the page or repository, and assume that they are looking at BioCro documentation. I don't think people will immediately recognize that "biocro-dev" is a different thing.

OK, the biocro-dev documentation site is cleaned out.  I put in automatic redirects for the landing page, the pkgdown home page, and the quantity docs page in case someone clicks a link in the README.  I did all this for the ebimodeling/biocro documentation site as well.

**Pull request merged**
